relay-server/src/services/health_check.rs

```diff
@@ -98,17 +113,37 @@ impl HealthCheckService {
                     return false;
                 }
 
-                self.project_cache
-                    .send(SpoolHealth)
+                let project_cache_sent = Instant::now();
+                let result = self
+                    .project_cache
+                    .send(SpoolHealthWrapper {
+                        message: SpoolHealth,
+                        sent_instant: project_cache_sent,
+                    })
                     .await
-                    .unwrap_or_default()
+                    .unwrap_or_default();
+                relay_statsd::metric!(
+                    timer(RelayTimers::HealthcheckServiceProjectcacheResponseDuration) =
+                        project_cache_sent.elapsed(),
+                );
+
+                result
             }
         }
     }
 
     async fn handle_message(&self, message: HealthCheck) {
+        let handle_start = Instant::now();
         let HealthCheck(message, sender) = message;
-        let response = self.handle_is_healthy(message).await;
+        relay_statsd::metric!(
+            timer(RelayTimers::HealthcheckServiceReceivedDelay) = message.received.elapsed(),
+        );
+
+        let response = self.handle_is_healthy(message.kind).await;
+        relay_statsd::metric!(
+            timer(RelayTimers::HealthcheckServiceHandleDuration) = handle_start.elapsed(),
```

Comment:
Do we need this dance with `Instant`?

Something like this is what we do everywhere else:
```rs
        metric!(
            timer(RelayTimers:: HealthcheckServiceReceivedDelay),
            message = ty, { self.handle_is_healthy(message.kind).await }
        )
```